examples/nvme/perf/perf.c

```diff
@@ -1072,19 +1115,23 @@ int main(int argc, char **argv)
 	g_tsc_rate = rte_get_timer_hz();
 
 	if (register_workers() != 0) {
-		return 1;
+		rc = -1;
+		goto FAIL;
```

Comment:
The label should be all-lowercase for consistency with the rest of the code.  I also think it would be clearer to call it "cleanup" or similar, since normal execution falls through that case as well.